packages/components/src/color-palette/style.scss

```diff
@@ -41,6 +41,7 @@ $border-as-box-shadow: inset 0 0 0 $border-width rgba(0, 0, 0, 0.2);
 		background-position: 0 0, 24px 24px;
 		/*rtl:end:ignore*/
 		background-size: calc(2 * 24px) calc(2 * 24px);
+		border-radius: $radius-medium - 1 $radius-medium - 1 0 0;
```

Comment:
If we find this approach difficult to understand, we can also solve the problem by applying `overflow: hidden` to the button itself.